packages/core/src/lib/waku_peer_exchange/index.ts

```diff
@@ -2,10 +2,12 @@ import type { Stream } from "@libp2p/interface-connection";
 import { ConnectionManager } from "@libp2p/interface-connection-manager";
 import { PeerId } from "@libp2p/interface-peer-id";
 import type { Peer, PeerStore } from "@libp2p/interface-peer-store";
+import { ENR } from "@waku/enr";
```

Comment:
`@waku/enr` brings a lot of dependencies such as cryptographic ones. I think it's best if we extract peer-exchange to own package, like dns discovery.

Reply:
Opened a follow-up PR for this: https://github.com/waku-org/js-waku/pull/1036